daphne/src/roles/mod.rs

```diff
@@ -1918,11 +1918,11 @@ mod test {
             task_expiration: t.now + 86400 * 14,
             vdaf_config: taskprov::VdafConfig {
                 dp_config: taskprov::DpConfig::None,
-                var: taskprov::VdafTypeVar::Prio3Aes128Count,
+                var: taskprov::VdafTypeVar::Prio2 { dimension: 10 },
             },
         }
         .get_encoded_with_param(&t.helper.global_config.taskprov_version.unwrap());
-        let taskprov_id = crate::taskprov::compute_task_id(
```

Comment:
My text editor highlights this in red, which suggests to me that `super` is preferred.